docs/docs/dev/security/certificates.md

```diff
@@ -28,102 +27,83 @@ Note, this CA is not the same CA for service-to-service communication.
+The data plane proxy Injector in the control plane automatically provides the CA to the Kuma DP sidecar 
+so Kuma DP can confirm the control plane identity.
 
-When configured using the `--tls-general-secret` and `--tls-general-ca-bundle` flags, Kuma will use the specified CA and certificate to protect data plane proxy to control plane traffic. Additionally, if not configured otherwise using additional options described below, Kuma will also use the specified CA and certificate to protect user to control plane traffic and control plane to control plane traffic when in multizone configurations.
 :::
 ::: tab "Kubernetes (HELM)"
-1) Prepare certificates
-
-Generate TLS pair with a PKI of your choice and store it in PEM-encoded format in `/tmp/tls.crt`, `/tmp/tls.key`. Store the CA that was used to sign this pair in `/tmp/ca.crt`
-
-We can also use `kumactl` to generate self-signed certs.
-```sh
-kumactl generate tls-certificate \
-  --type=server \
-  --hostname=kuma-control-plane.kuma-system \ # adjust the name if you are installing Kuma to other namespace
-  --hostname=kuma-control-plane.kuma-system.svc \ # adjust the name if you are installing Kuma to other namespace
-  --cert-file=/tmp/tls.crt \
-  --key-file=/tmp/tls.key
-cp /tmp/tls.crt /tmp/ca.crt # since this is self-signed cert, the cert is also a CA
-```
-
-2) Create a secret in the namespace where the control plane is installed
+Create a secret in the namespace where the control plane is installed:
 ```sh
-kubectl create secret generic general-tls-certs -n kuma-system \
+kubectl create secret generic general-tls-certs -n <namespace> \
   --from-file=tls.crt=/tmp/tls.crt \
   --from-file=tls.key=/tmp/tls.key \
   --from-file=ca.crt=/tmp/ca.crt
 ```
 
-3) Point to this secret when installing Kuma
-
-Set `controlPlane.tls.general.secretName` to `general-tls-certs` and `controlPlane.tls.general.caBundle` to `<BASE64 content of ca.crt>`
+Point to this secret when installing Kuma:
+```sh
+helm install --create-namespace --namespace <namespace> kuma kuma/kuma \
+  --set controlPlane.tls.general.secretName=general-tls-certs \
+  --set controlPlane.tls.general.caBundle=$(cat /tmp/ca.crt | base64)
+```
 
-The data plane proxy Injector in the control plane automatically provides the CA to the Kuma DP sidecar so Kuma DP can confirm the control plane identity.
+The data plane proxy Injector in the control plane automatically provides the CA to the Kuma DP sidecar 
+so Kuma DP can confirm the control plane identity.
 
-When configured using the `controlPlane.tls.general.secretName` and `controlPlane.general.caBundle` values in Helm, Kuma will use the specified CA and certificate to protect data plane proxy to control plane traffic. Additionally, if not configured otherwise using options described below, Kuma will also use the specified CA and certificate to protect user to control plane traffic and control plane to control plane traffic when in multizone configurations.
```

Comment:
I think that was added by Scott, because it was not obvious what general cert is.

Reply:
Maybe it should be higher up in the explaination?

Reply:
I added a tip like this:
<img width="773" alt="image" src="https://user-images.githubusercontent.com/12110261/175921867-c23c5f08-0194-4538-be8d-583beb7e6a5d.png">